### ISSUE TITLE
Log every failed Github API call because of quota

### ISSUE THREAD

**Issue by g4s8** [zerocracy/farm#1390]
We are using `Quota` class to check Github API quota limit, but almost always we just skip API calls if we over the quota (using `return` in stakeholders). Let's log all quota related issues with `WARNING` level to send all these messages to Sentry as warnings. Looks like we sometimes skip important Github notifications because of that.

**Comment by 0crat** [comment]
Job [#1390](https://github.com/zerocracy/farm/issues/1390) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by g4s8** [comment]
@0crat assign @rgevaerd

**Comment by 0crat** [comment]
> @0crat assign @rgevaerd ([here](https://github.com/zerocracy/farm/issues/1390#issuecomment-408623348))

@g4s8 The job [#1390](https://github.com/zerocracy/farm/issues/1390) assigned to @rgevaerd[/z](https://www.0crat.com/u/rgevaerd), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2091746663); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); we should be aware that @rgevaerd[/z](https://www.0crat.com/u/rgevaerd) is on vacation; this ticket may be delayed; there will be a monetary reward for this job

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by krzyk** [comment]
@g4s8 There are three places where over quota is checked:
1. AcceptInvitations - here we just `return`
2. notify_in_github - we copy claim and add until 5 seconds (so it will be retried)
3. TkGithub - we return 503, so rehttp should retry it

So is it correct to only fix AcceptInvitations? Are only invitations lost here?

**Comment by g4s8** [comment]
@krzyk I think we need to collect all quota related issues, even in `notify_in_github` to be able to count how often this situation happens.

**Comment by rgevaerd** [comment]
@0crat waiting on PR #1501

**Comment by 0crat** [comment]
> @0crat waiting on PR #1501 ([here](https://github.com/zerocracy/farm/issues/1390#issuecomment-409056544))

@rgevaerd The impediment for [#1390](https://github.com/zerocracy/farm/issues/1390) was registered successfully by @rgevaerd[/z](https://www.0crat.com/u/rgevaerd)

**Comment by 0pdd** [comment]
@g4s8 2 puzzles [#1547](https://github.com/zerocracy/farm/issues/1547), [#1548](https://github.com/zerocracy/farm/issues/1548) are still not solved.

**Comment by rgevaerd** [comment]
@g4s8 #1501 has been merged, can we close this now?

**Comment by rgevaerd** [comment]
@g4s8 ping

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @rgevaerd[/z](https://www.0crat.com/u/rgevaerd), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1390](https://github.com/zerocracy/farm/issues/1390) is now out of scope

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @rgevaerd[/z](https://www.0crat.com/u/rgevaerd)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0pdd** [comment]
@g4s8 2 puzzles [#unknown](), [#unknown]() are still not solved.
